### PR TITLE
Update network troubleshooting docs for naming changes

### PR DESCRIPTION
**Pull request by danwinship**
Update for https://github.com/openshift/cluster-network-operator/pull/68

### CODE REVIEW

File: docs/user/troubleshooting.md [hunk 1]
```diff
@@ -244,13 +244,13 @@ If it doesn't exist, the installer didn't create it. You'll have to run `openshi
...
 ```sh
-kubectl -n openshift-cluster-network-operator get pods
+kubectl -n openshift-network-operator get pods
 ```
 
 And retrieve the logs. Note that, on multi-master systems, the operator will perform leader election and all other operators will sleep:
 
 ```sh
-kubectl -n openshift-cluster-network-operator logs -l "k8s-app=cluster-network-operator"
+kubectl -n openshift-network-operator logs -l "name=network-operator"
```
Comment (abhinavdahiya) [inline docs/user/troubleshooting.md]: @danwinship can you confirm this change of label selector works. 😇

Comment (danwinship) [reply docs/user/troubleshooting.md]:
```
danw@p50:~> kubectl -n openshift-network-operator logs -l "name=network-operator" | tail -5
2019/01/29 17:16:48 reconciling (/v1, Kind=ConfigMap) openshift-sdn/sdn-config
2019/01/29 17:16:48 update was noop
2019/01/29 17:16:48 reconciling (apps/v1, Kind=DaemonSet) openshift-sdn/sdn
2019/01/29 17:16:49 update was noop
2019/01/29 17:16:49 all objects successfully applied
```

**Comment by wking** [comment]
> Update for openshift/cluster-network-operator#68

I [still][1] think these docs should live in the network-operator repository (where we could link to them), so they could evolve alongside the code they describe ;).

[1]: https://github.com/openshift/installer/pull/669/#pullrequestreview-175390535

**Pull request merged**